Socket_Adapter/CRUD/Pull.cs

```diff
@@ -37,8 +37,12 @@ public override IEnumerable<object> Pull(IRequest request, Dictionary<string, ob
                 else
                     return new List<object>();
             }
-
-            return new List<object>();
+            else
+            {
+                Engine.Reflection.Compute.RecordWarning("The socket adater only supports FilterRequests.");
```

Comment:
Is this correct? 
I thought that the `SocketAdapter` did not support `FilterRequest`s: https://github.com/BHoM/Socket_Toolkit/issues/52

Or maybe is the issue superseeded?

ps. there is also a type, `adater` for `adapter`

Reply:
Ah sorry - I see now. It does support `FilterRequest`s, but filters only by `Tag`. It may be worth adding it.
```suggestion
Engine.Reflection.Compute.RecordWarning("The socket adapter only supports FilterRequest with Tags.")
```